src/github3/pulls.py

```diff
@@ -575,6 +576,15 @@ def reviews(self, number=-1, etag=None):
         url = self._build_url('reviews', base_url=self._api)
         return self._iter(int(number), url, PullReview, etag=etag)
 
+    def combined_status(self):
+        """ Retrieve the combined status for this Pull Request
```

Comment:
a combined status refers to a particular Head of a pull request, not the pull request itself. It's a weird thing, but important distinction. I suggest updating the doc string to make that clear. Perhaps:
```suggestion
        """ Retrieve the combined status for the Head of this Pull Request
```